src/lib/merkle_ledger/database.ml

```diff
@@ -430,14 +428,18 @@ module Make (Inputs : Inputs_intf) :
   let last_filled t = Account_location.last_location t
 
   let token_owners t =
-    Token_owner.get_all t
+    Tokens.Owner.get_all t
     |> List.map ~f:(fun (tid, pk) -> Account_id.create pk tid)
     |> Account_id.Set.of_list
 
-  let token_owner = Token_owner.get
+  let token_owner = Tokens.Owner.get
 
   let tokens = Tokens.get
 
+  let next_available_token = Tokens.next_available
```

Comment:
you could use `%%define_locally` for these 2 definitions (but I don't think you like that)